### PR TITLE
PROD-3631 - add hover state for about us

### PR DESCRIPTION
**Pull request by vas3a**
https://topcoder.atlassian.net/browse/PROD-3631

- moves "About us" in the marketing menu items.
- adds description for "About us"

### CODE REVIEW

**Review by testflyjets — APPROVED** [review]

**Review by brooketopcoder — APPROVED** [review]

**Comment by brooketopcoder** [comment]
This looks good, but there are going to be conflicts w/the PRs that are unifying the dev/prod configs.  So let's hold off on merging this until we can get those changes merged to dev.

**Comment by vas3a** [comment]
@testflyjets I'm merging this one, because the blocking PR has already been merged.

**Pull request merged**
